nrf52-hal-common/Cargo.toml

```diff
@@ -19,6 +19,7 @@ edition = "2018"
 [dependencies]
 cortex-m = "0.5.8"
 nb = "0.1.1"
+rand = { version = "0.6.5", default-features = false }
```

Comment:
Could you please use the `[dependencies.rand]` syntax here for consistency?

Reply:
Done :)